cplugin/v2/fromproto/destination_test.go

```diff
@@ -0,0 +1,149 @@
+// Copyright © 2024 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fromproto
+
+import (
+	"testing"
+
+	"github.com/conduitio/conduit-commons/opencdc"
+	opencdcv1 "github.com/conduitio/conduit-commons/proto/opencdc/v1"
+	"github.com/conduitio/conduit-connector-protocol/cplugin"
+	connectorv2 "github.com/conduitio/conduit-connector-protocol/proto/connector/v2"
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/go-cmp/cmp/cmpopts"
+	"github.com/matryer/is"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+var (
+	testRecordGo = opencdc.Record{
+		Position:  opencdc.Position("standing"),
+		Operation: opencdc.OperationUpdate,
+		Metadata:  opencdc.Metadata{"foo": "bar"},
+		Key:       opencdc.RawData("padlock-key"),
+		Payload: opencdc.Change{
+			Before: opencdc.RawData("yellow"),
+			After: opencdc.StructuredData{
+				"bool": true,
+
+				"int":   float64(1),
+				"int32": float64(1),
+				"int64": float64(1),
+
+				"float32": float64(1.5),
+				"float64": float64(1.5),
```

Comment:
Shouldn't the values be int(1), int32(1), etc.?

Reply:
They "should", they are not 👎 We lose the types in standalone connectors. It's essentially the same encoding as if we were sending JSON. All the more reason to lean into schemas.

Reply:
Ah, right, this is a fromproto test!